src/components/combobox-item/combobox-item.tsx

```diff
@@ -126,6 +126,13 @@ export class ComboboxItem implements ConditionalSlotComponent, InteractiveCompon
    */
   @Event({ cancelable: false }) calciteComboboxItemChange: EventEmitter<DeprecatedEventPayload>;
 
+  /**
+   * Emits whenever the component is focused.
+   *
+   * **Note:**: The event's payload is deprecated, please use the event's `target`/`currentTarget` instead
+   */
+  @Event({ cancelable: false }) calciteComboboxItemFocus: EventEmitter<DeprecatedEventPayload>;
```

Comment:
@alisonailea Do you think we could call this event `calciteInternalComboboxItemChange` instead? I don't think we want to make a public focus event for this fix.

Curious, If this is a new event, why is the payload deprecated?